main.go

```diff
@@ -156,7 +153,9 @@ func main() {
 		log.Fatalln("Error:", err)
 	}
 
-	if err := ioutil.WriteFile(filepath.Join(basePath, ".version"), []byte(version), 0660); err != nil {
+	var b bytes.Buffer
+	PrintVersion(&b)
+	if err := ioutil.WriteFile(filepath.Join(basePath, "version"), b.Bytes(), 0660); err != nil {
```

Comment:
oh yes, I was thinking of: https://github.com/feedhenry/fh-system-dump-tool/blob/master/openshift_metadata.go

but I see now that is only for openshift metadata, not general metadata.